test/e2e/specs/AutolendingPage.spec.js

```diff
@@ -303,12 +303,19 @@ describe('Autolending Page Spec', () => {
 
 			// Visit autolending settings
 			cy.visit('/settings/autolending');
+			// Show the modal
```

Comment:
Fixed these tests since now we have to pop up a modal, then hit the save button in the modal to check our changes.